docs/projects/template-fps/tutorial.md

```diff
@@ -197,7 +197,7 @@ You can regenerate the `default.snapshot` file from the SpatialOS menu option in
 
 While they are human-readable you are able to manually edit the values of the properties within, however be careful not to make mistakes that will inhibit the conversion back to binary form!<%(/Expandable)%>
```

Comment:
While they are human-readable you are able to manually edit the values of the properties within,

->

While they are human-readable and you can manually edit the values of the properties within,